lib/active_merchant/billing/gateways/merchant_e_solutions.rb

```diff
@@ -55,10 +57,11 @@ def capture(money, transaction_id, options = {})
       end
 
       def store(creditcard, options = {})
-        post = {}
-        post[:client_reference_number] = options[:customer] if options.has_key?(:customer)
-        add_creditcard(post, creditcard, options)
-        commit('T', nil, post)
+        response = MultiResponse.run do |r|
+          r.process { temporary_store(creditcard, options) }
+          r.process { verify(creditcard, { store_card: 'y' }) }
+        end
+        response.responses.last
```

Comment:
I am pretty sure that MultiResponse uses the last response by default (because `use_first_response` [defaults to false](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/response.rb#L49)).

So if you take away the assignment to the `response` variable then `MultiResponse.run` should just return the last response in the process.

Reply:
you're right, it responds with the last response so removing this unnecessary line.